### PR TITLE
fix(IDLIX): update domain

### PR DESCRIPTION
**Pull request by skullysmods**
## Description 
Update domain 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

</details>

### CODE REVIEW

File: websites/I/IDLIX/metadata.json [hunk 1]
```diff
@@ -5,13 +5,19 @@
 		"id": "638080361179512853",
 		"name": "Dark_Ville"
 	},
+	"contributors": [
+		{
+			"name": "Atom Skully",
+			"id": "671037171611729920"
+		}
+	],
```
Comment (darkvillager2) [inline websites/I/IDLIX/metadata.json:13]: Not enough contribution for this.

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

File: websites/I/IDLIX/metadata.json [hunk 2]
```diff
@@ -5,13 +5,19 @@
...
 	"service": "IDLIX",
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "vip.idlixofficial.net",
-	"regExp": "([0-9]+)(?<!10)[.]([0-9]+)(?<!192[.]168)(?<!172[.](1[6-9]|2[0-9]|3[0-1]))[.]([0-9]+)[.]([0-9]+)|(((tv([0-9]?))?(vip)?[.])?id(f)?lix(official)?[.][a-z]{2,6})",
-	"version": "1.1.2",
+	"url": "vip.idlixofficialx.net",
+	"regExp": "([0-9]+)(?<!10)[.]([0-9]+)(?<!192[.]168)(?<!172[.](1[6-9]|2[0-9]|3[0-1]))[.]([0-9]+)[.]([0-9]+)|(((tv([0-9]?))?(vip)?[.])?id(f)?lix(officialx)?[.][a-z]{2,6})",
```
Comment (darkvillager2) [inline websites/I/IDLIX/metadata.json:19]:
```suggestion
	"regExp": "([0-9]+)(?<!10)[.]([0-9]+)(?<!192[.]168)(?<!172[.](1[6-9]|2[0-9]|3[0-1]))[.]([0-9]+)[.]([0-9]+)|(((tv([0-9]?))?(vip)?[.])?id(f)?lix(official(x)?)?[.][a-z]{2,6})",
```

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

File: websites/I/IDLIX/metadata.json (hunk 1 above)
Comment (skullysmods) [reply websites/I/IDLIX/metadata.json]: Okay, I changed it, no worries

**Review by darkvillager2 — APPROVED** [review]
lgtM

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**